sdk/eventhub/eventhubs-checkpointstore-blob/test/snippets.spec.ts

```diff
@@ -1,14 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { env } from "process";
-import * as dotenv from "dotenv";
-
 // if you modify these imports update the imports in the snippet below as well
 // (JS compatible there)
+/*
 import { EventHubConsumerClient } from "@azure/event-hubs";
 import { ContainerClient } from "@azure/storage-blob";
 import { BlobCheckpointStore } from "../src";
+*/
```

Comment:
These imports have been sorted at line 12 so is there a need to comment them here?

If not, let's delete them:
```suggestion
```